parts/k8s/kubernetesmasterresources.t

```diff
@@ -1,14 +1,16 @@
 {{if .MasterProfile.IsManagedDisks}}
     {
-      "apiVersion": "[variables('apiVersionStorageManagedDisks')]",
+      "apiVersion": "[variables('apiVersionDefault')]",
       "location": "[variables('location')]",
       "name": "[variables('masterAvailabilitySet')]",
       "properties":
-        {
-            "platformFaultDomainCount": 2,
-            "platformUpdateDomainCount": 3,
-		        "managed" : true
-        },
+      {
+        "platformFaultDomainCount": 2,
+        "platformUpdateDomainCount": 3
+      },
+      "sku": {
+        "name": "Aligned"
```

Comment:
The only significant change in templates in this PR: `managed` is now deprecated in favor of `Aligned` sku for availability sets with managed disks 

https://docs.microsoft.com/en-us/azure/templates/microsoft.compute/availabilitysets

Reply:
This change is simply to fulfill the new interface defined by the new API version, right? (i.e., no functional change there)

Reply:
It should be from what I understand. That's something we should verify and be 100% confident about before we commit this change.